jbpm-wb-common/jbpm-wb-common-client/src/test/java/org/jbpm/workbench/common/client/dataset/DataSetAwareSelectTest.java

```diff
@@ -113,18 +124,75 @@ public void testLookupDataSet() throws Exception {
         final DataSetLookup lookup = mock(DataSetLookup.class);
         when(lookup.getDataSetUUID()).thenReturn(dataUUID);
         dataSetAwareSelect.setDataSetLookup(lookup);
-        dataSetAwareSelect.setValueColumnId("value");
-        dataSetAwareSelect.setTextColumnId("test");
+        dataSetAwareSelect.setValueColumnId(VALUE_COLUMN_ID);
+        dataSetAwareSelect.setTextColumnId(TEXT_COLUMN_ID);
 
         doAnswer(new Answer() {
             @Override
             public Void answer(InvocationOnMock invocation) throws Throwable {
                 final DataSet dataSet = mock(DataSet.class);
                 when(dataSet.getRowCount()).thenReturn(1);
                 when(dataSet.getValueAt(0,
-                                        "value")).thenReturn(columnValue);
+                                        VALUE_COLUMN_ID)).thenReturn(columnValue);
+                when(dataSet.getValueAt(0,
+                                        TEXT_COLUMN_ID)).thenReturn(columnText);
+                ((DataSetReadyCallback) invocation.getArguments()[1]).callback(dataSet);
+                return null;
+            }
+        }).when(dataSetClientServices).lookupDataSet(any(DataSetLookup.class),
+                                                     any(DataSetReadyCallback.class));
+
+        dataSetAwareSelect.onDataSetReady(new DataSetReadyEvent(filterSettings));
+
+        final ArgumentCaptor<DataSetLookup> captor = ArgumentCaptor.forClass(DataSetLookup.class);
+        verify(dataSetClientServices).lookupDataSet(captor.capture(),
+                                                    any(DataSetReadyCallback.class));
+        assertTrue(captor.getValue() instanceof ConsoleDataSetLookup);
+        final ConsoleDataSetLookup cdsl = (ConsoleDataSetLookup) captor.getValue();
+        assertEquals(serverTemplateId,
+                     cdsl.getServerTemplateId());
+        assertEquals(dataUUID,
+                     cdsl.getDataSetUUID());
+        verify(select).addOption(columnText,
+                                 columnValue);
+        verify(select).enable();
+        verify(select).removeAllOptions();
+    }
+
+    @Test
+    public void testLookupDataSetEmptyString() throws Exception {
```

Comment:
Would only suggest renaming the method name to ```testLookupDataSetEmptyStringOrNull()```

Reply:
Done